includes/class-wc-payments-account.php

```diff
@@ -1982,6 +1985,7 @@ private function init_stripe_onboarding( string $setup_mode, string $wcpay_conne
 		}
 
 		$self_assessment_data = isset( $_GET['self_assessment'] ) ? wc_clean( wp_unslash( $_GET['self_assessment'] ) ) : [];
+		$capabilities         = isset( $_GET['capabilities'] ) ? json_decode( wc_clean( wp_unslash( $_GET['capabilities'] ) ), true ) : [];
```

Comment:
See that json_decode will return `null` if it fails to decode. You should check for that and put in an empty array.

Reply:
Good catch, fixed it.